tests/langchain_handlers/test_langchain_handler.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
```

Comment:
can we use pytest here with assertions. otherwise automated testing will always pass

Reply:
leaving for now